lib/zipint.h

```diff
@@ -486,6 +486,8 @@ typedef struct _zip_pkware_keys zip_pkware_keys_t;
 #endif
 #endif
 
+#define zip_realloc(inout_memory, inout_alloced, element_size, additional_elements, error) zip_realloc_implementation((void **)inout_memory, inout_alloced, element_size, additional_elements, error)
```

Comment:
I was thinking about dropping the `element_size` parameter from the macro and make the macro to pass `(sizeof(**(inout_memory)))` instead, but this is probably too much magic.